source/tyr/readme.md

```diff
@@ -740,52 +740,110 @@ If nothing if configured, default POI are extracted. At this time the list is as
-            "name": "foo\u00e9",
-            "uri": "foo:baff"
-        }
-    ]
+  "poi_types": [
+    {"id": "amenity:college", "name": "École"},
+    {"id": "amenity:university", "name": "Université"},
+    {"id": "amenity:theatre", "name": "Théâtre"},
+    {"id": "amenity:hospital", "name": "Hôpital"},
+    {"id": "amenity:post_office", "name": "Bureau de poste"},
+    {"id": "amenity:bicycle_rental", "name": "Station VLS"},
+    {"id": "amenity:bicycle_parking", "name": "Parking vélo"},
+    {"id": "amenity:parking", "name": "Parking"},
+    {"id": "amenity:police", "name": "Police, gendarmerie"},
+    {"id": "amenity:townhall", "name": "Mairie"},
+    {"id": "leisure:garden", "name": "Jardin"},
+    {"id": "leisure:park", "name": "Parc, espace vert"}
+  ],
+  "rules": [
+    {
+      "osm_tags_filters": [
+        {"key": "amenity", "value": "college"}
+      ],
+      "poi_type_id": "amenity:college"
+    },
+    {
+      "osm_tags_filters": [
+        {"key": "amenity", "value": "university"}
+      ],
+      "poi_type_id": "amenity:university"
+    },
+    {
+      "osm_tags_filters": [
+        {"key": "amenity", "value": "theatre"}
+      ],
+      "poi_type_id": "amenity:theatre"
+    },
+    {
+      "osm_tags_filters": [
+        {"key": "amenity", "value": "hospital"}
+      ],
+      "poi_type_id": "amenity:hospital"
+    },
+    {
+      "osm_tags_filters": [
+        {"key": "amenity", "value": "post_office"}
+      ],
+      "poi_type_id": "amenity:post_office"
+    },
+    {
+      "osm_tags_filters": [
+        {"key": "amenity", "value": "bicycle_rental"}
+      ],
+      "poi_type_id": "amenity:bicycle_rental"
+    },
+    {
+      "osm_tags_filters": [
+        {"key": "amenity", "value": "bicycle_parking"}
+      ],
+      "poi_type_id": "amenity:bicycle_parking"
+    },
+    {
+      "osm_tags_filters": [
+        {"key": "amenity", "value": "parking"}
+      ],
+      "poi_type_id": "amenity:parking"
+    },
+    {
+      "osm_tags_filters": [
+        {"key": "amenity", "value": "police"}
+      ],
+      "poi_type_id": "amenity:police"
+    },
+    {
+      "osm_tags_filters": [
+        {"key": "amenity", "value": "townhall"}
+      ],
+      "poi_type_id": "amenity:townhall"
+    },
+    {
+      "osm_tags_filters": [
+        {"key": "leisure", "value": "garden"}
+      ],
+      "poi_type_id": "leisure:garden"
+    },
+    {
+      "osm_tags_filters": [
+        {"key": "leisure", "value": "park"}
+      ],
+      "poi_type_id": "leisure:park"
+    }
+  ]
 }
-
 ```
 
-If you want to add a poi type for an instance you have to POST it:
-
-    curl 'http://localhost:5000/v0/instances/fr-bre/poi_types/foo:baff?name=foo' -X POST
-
-The name of a poi type is optional and can also be passed in a json like this:
-
-    curl 'http://localhost:5000/v0/instances/fr-bre/poi_types/foo:json' -X POST -H 'content-type: application/json' -d '{"name": "json"}'
+If you want to add poi types for an instance you have to POST it:
```

Comment:
For memory (to be changed in my PR also): "you have to POST again all the json (previous AND new poi types)"